site/public/css/polls.css

```diff
@@ -39,10 +39,28 @@
     cursor: pointer;
 }
 
-.response-btn {
+.option-btn {
+    vertical-align: middle;
+}
+
+.option-btn input[type="radio"]{
     vertical-align: top;
 }
 
+.option-btn input[type="checkbox"]{
+    transform: scale(0.8);
+}
```

Comment:
Is this still needed now that everything is in a table?

Reply:
Yes, a checkbox in the page is way too large comparing to a radio button. I tried to retain as much of the original size but still keep it inline with the size of a row of text and all that. This should over make it to have a nicer look but does not break things since it is scaled down not up. The functionality should be tested throughout though, using different browsers.

Reply:
Yes, the checkboxes are larger than most radio buttons, but I don't see that as a problem.  Different systems are going to show them differently, and rescaling things here just makes this page different from the rest of Submitty.  I'm personally of the opinion that the checkbox size should just be left alone, but it's a rather subjective argument.

Reply:
In the functionality test, the other guys also tested the scaled down version of the checkbox in this page. It should be good to apply here - because they're really strangely big. In main it is also the source of breaking lines of text of an answer field because its size is adding extra spacing between the first row and second row of text. 

But that problem is solved by putting it in a table cell and centering it vertically. I think it is good either way too. I'm removing the change.